### ISSUE TITLE
Add a configurable description to be added to the swagger ui

### ISSUE THREAD

**Issue by p013570** [gchq/Gaffer#1419]
This will help to brand the REST API Swagger UI for different systems. 
The description should be configurable from the rest.properties file.

**Comment by p013570** [comment]
This should probably be done after #1420.

**Comment by p013570** [comment]
Merged into develop
